### ISSUE TITLE
Javascript Error in Background

### ISSUE THREAD

**Issue by TehTotalPwnage** [crmarsh/discord-linux-bugs#60]
I'm actually not sure what caused this issue, as I just left DiscordCanary running in the background. Anyways, here's the issue:

```
Uncaught Exception:
Error: Attempting to call a function in a renderer window that has been closed or released. Function provided here: d98158f5ce2e2dd38871.js:40:12025.
    at Function.callIntoRenderer (/usr/share/discord-canary/resources/electron.asar/browser/rpc-server.js:198:19)
```

Aside from that, it doesn't look like DiscordCanary actually crashed or any issues appeared with it. Just the notification. However, the notification does keep reappearing every time I attempt to dismiss it. I'm running on Ubuntu 14.04 LTS.

**Comment by zamabe** [comment]
Same problem here, different "Function provided here".

```
Uncaught Exception:
Error: Attempting to call a function in a renderer window that has been closed or released. Function provided here: 1ab275159396d96456c7.js:51:1668.
    at Function.callIntoRenderer (/usr/share/discord-canary/resources/electron.asar/browser/rpc-server.js:198:19)
```

**Comment by crmarsh** [comment]
I think this should be better and/or fixed in 0.0.9
